### DISCUSSION TITLE
I can't login to the Shimmer 2.0.0 app

### DISCUSSION THREAD

**Discussion by safi1972** [iotaledger/firefly#4899] [Help & Support]
I can't login to the Shimmer 2.0.0 app. App says "something went wrong" I have 2100 SMR there. What should I do?
![shimmer](https://user-images.githubusercontent.com/115795128/196118103-d21d45b1-09a4-44bc-a449-bfe245f531a1.png)

**Comment by maxwellmattryan** [comment]
Hey @safi1972, can you check the "Error log" from the top menu?

**Reply by safi1972** [reply]
This will probably be the problem. But what about that?


Fri, 07 Oct 2022 09:15:36 GMT
BaseError: An error occurred while searching for rewards

**Reply by safi1972** [reply]
I updated the app to 2.0.1 and there is a new error message
Thu, 20 Oct 2022 07:35:33 GMT
iotaClientError: response error with status code 400: {"error":{"code":"400","message":"invalid parameter, error: invalid bech32 address, expected prefix: smr: code=400, message=invalid parameter"}}
, URL: https://api.shimmer.network/api/indexer/v1/outputs/basic?address=rms1qqjnk9c5qxujk33j5spyc9cest3nkwjetwvlhfytrr9d2rdl4s0zsyfyq7m

Thu, 20 Oct 2022 07:30:48 GMT
iotaClientError: response error with status code 400: {"error":{"code":"400","message":"invalid parameter, error: invalid bech32 address, expected prefix: smr: code=400, message=invalid parameter"}}
, URL: https://api.shimmer.network/api/indexer/v1/outputs/basic?address=rms1qqjnk9c5qxujk33j5spyc9cest3nkwjetwvlhfytrr9d2rdl4s0zsyfyq7m

Thu, 20 Oct 2022 07:29:37 GMT
iotaClientError: response error with status code 400: {"error":{"code":"400","message":"invalid parameter, error: invalid bech32 address, expected prefix: smr: code=400, message=invalid parameter"}}
, URL: https://api.shimmer.network/api/indexer/v1/outputs/basic?address=rms1qqjnk9c5qxujk33j5spyc9cest3nkwjetwvlhfytrr9d2rdl4s0zsyfyq7m

Thu, 20 Oct 2022 07:26:48 GMT
iotaClientError: response error with status code 400: {"error":{"code":"400","message":"invalid parameter, error: invalid bech32 address, expected prefix: smr: code=400, message=invalid parameter"}}
, URL: https://api.shimmer.network/api/indexer/v1/outputs/basic?address=rms1qqjnk9c5qxujk33j5spyc9cest3nkwjetwvlhfytrr9d2rdl4s0zsyfyq7m

Thu, 20 Oct 2022 07:26:12 GMT
iotaClientError: response error with status code 400: {"error":{"code":"400","message":"invalid parameter, error: invalid bech32 address, expected prefix: smr: code=400, message=invalid parameter"}}
, URL: https://api.shimmer.network/api/indexer/v1/outputs/basic?address=rms1qqjnk9c5qxujk33j5spyc9cest3nkwjetwvlhfytrr9d2rdl4s0zsyfyq7m

Thu, 20 Oct 2022 07:13:55 GMT
undefined: undefined

Mon, 17 Oct 2022 11:40:00 GMT
undefined: undefined

Mon, 17 Oct 2022 06:47:51 GMT
undefined: undefined

Mon, 17 Oct 2022 06:47:21 GMT
undefined: undefined

Mon, 17 Oct 2022 06:31:21 GMT
undefined: undefined

Sat, 15 Oct 2022 14:39:25 GMT
undefined: undefined

Sat, 15 Oct 2022 14:38:01 GMT
undefined: undefined

Sat, 15 Oct 2022 14:30:12 GMT
undefined: undefined

Fri, 14 Oct 2022 12:19:56 GMT
undefined: undefined

Fri, 14 Oct 2022 11:58:28 GMT
undefined: undefined

Fri, 14 Oct 2022 11:43:29 GMT
undefined: undefined

Fri, 14 Oct 2022 11:42:32 GMT
undefined: undefined

Fri, 14 Oct 2022 11:41:48 GMT
undefined: undefined

Wed, 12 Oct 2022 09:46:26 GMT
undefined: undefined

Wed, 12 Oct 2022 07:40:17 GMT
undefined: undefined

Wed, 12 Oct 2022 07:34:02 GMT
undefined: undefined

Wed, 12 Oct 2022 06:46:59 GMT
undefined: undefined

Wed, 12 Oct 2022 06:22:00 GMT
undefined: undefined

Tue, 11 Oct 2022 17:49:13 GMT
undefined: undefined

Tue, 11 Oct 2022 17:47:45 GMT
undefined: undefined

Sat, 08 Oct 2022 11:01:45 GMT
undefined: undefined

Sat, 08 Oct 2022 06:15:01 GMT
undefined: undefined

Fri, 07 Oct 2022 09:16:05 GMT
undefined: undefined

Fri, 07 Oct 2022 09:15:36 GMT
BaseError: An error occurred while searching for rewards

Fri, 07 Oct 2022 08:43:42 GMT
undefined: undefined

Fri, 07 Oct 2022 06:18:07 GMT
undefined: undefined

Fri, 07 Oct 2022 06:14:39 GMT
undefined: undefined

Fri, 07 Oct 2022 06:14:06 GMT
undefined: undefined

Fri, 07 Oct 2022 06:13:55 GMT
undefined: undefined
